components/accelerator/nvidia/info/component.go

```diff
@@ -145,7 +145,8 @@ func (c *component) Check() components.CheckResult {
 	if err != nil {
 		cr.err = err
 		cr.health = apiv1.HealthStateTypeUnhealthy
-		cr.reason = fmt.Sprintf("error getting device count: %s", err)
+		cr.reason = "error getting device count"
```

Comment:
Shall we delete the component? There is same data in MachineInfo.

Reply:
Useful for `gpud scan` :)

I can create a separate PR to replace it with machine info.

Reply:
`gpud scan` should rely on MachineInfo.